model-server/src/test/kotlin/org/modelix/model/server/ReplicatedRepositoryTest.kt

```diff
@@ -329,6 +329,7 @@ class ReplicatedRepositoryTest {
         println("all successful")
     }
 
+    @Ignore
```

Comment:
Maybe a [`@Disabled`](https://junit.org/junit5/docs/current/api/org.junit.jupiter.api/org/junit/jupiter/api/Disabled.html) with a description for why this is actually disabled would be a better fit. If kotlin test shall remain the target, a comment would be good.